Description: Debian: Degrade getopt gracefully without gettext
 python3.X-minimal includes getopt but not gettext. Use a fallback noop
 gettext, if it can't be imported.

Forwarded: not-needed

--- a/Lib/getopt.py
+++ b/Lib/getopt.py
@@ -30,7 +30,15 @@
 __all__ = ["GetoptError","error","getopt","gnu_getopt"]
 
 import os
-from gettext import gettext as _
+try:
+    from gettext import gettext as _
+except ImportError:
+    # Debian shims, only used by python3.X-minimal when the full stdlib is not
+    # installed
+    def gettext(message):
+        return message
+    _ = gettext  # avoid the definition above looking like a translated string
+    del gettext
 
 
 class GetoptError(Exception):
